frontend/src/components/AdminSettings.vue

```diff
@@ -209,6 +269,13 @@ const now = ref<Date>(new Date());
 const keycloakAdminRealmURL = ref<string>();
 const onFetchError = ref<Error | null>();
 const errorOnFetchingUpdates = ref<boolean>(false);
+const trustLevel = ref<number>(0);
+const maxChainLength = ref<number>(0);
```

Comment:
use the same names as in the database and backend (i.e. `wotMaxDepth` and `wotIdVerifyLen`) to avoid confusion. Especially since we have a trust level in other places, which has a completely different meaning.